trackData/frontend.json

```
@@ -61,6 +61,8 @@
         ],
         "examples": [
             "Implemented a way to steer away from our Jquery/OOUI and enable us to work with a more modern framework",
+            "Architected the new SSR with Vue.js on the wikidata team",
```

My first reaction was that this sounds oddly specific compared to the other examples. If the idea is to look at those and go "oh, I've done that so I fall into this category!" then this example has very limited reach. It's also quite backend-y for a frontend example depending on how you want to look at it.

```suggestion
            "Understands the challenges of server-side rendering for modern frameworks like Vue or React, and is able develop sustainable solutions for them",
```
^ is my attempt at a generalization of the example. "sustainable" is very fluffy, but I somehow wanted to convey that there are multiple solutions  - thinking of how we tried the php renderer, no ssr, trying to reduce template duplication, ... - and that the trick is to find good trade-offs, as usual :)